### PR TITLE
Update java-packages.csv to include Digital Twins

### PR DESCRIPTION
**Pull request by samvaity**
This PR adds newer client libraries of Digital Twins service to track in releases.
https://azure.github.io/azure-sdk-for-java/communication.html

### CODE REVIEW

**Comment by jongio** [comment]
Do we need to do for other langs as well?

**Review by czubair — APPROVED** [review]

**Comment by czubair** [comment]
> Do we need to do for other langs as well?

yes, pinging the other release drivers to follow suit.

**Comment by weshaggard** [comment]
I've already added the Communication packages in PR https://github.com/Azure/azure-sdk/pull/1837

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -1,5 +1,6 @@
 "Package","GroupId","VersionGA","VersionPreview","DisplayName","ServiceName","RepoPath","MSDocs","GHDocs","Type","Hide","Notes"
 "azure-ai-anomalydetector","com.azure","","3.0.0-beta.1","Anomaly Detector","Anomaly Detector","anomalydetector","","","client","",""
+"azure-digitaltwins-core","com.azure","","1.0.0-beta.1","Digital Twins","Digital Twins","digitaltwins","","","client","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:3]:
```suggestion
"azure-digitaltwins-core","com.azure","","1.0.0-beta.1","Digital Twins","Digital Twins","digitaltwins","NA","","client","",""
```

**Review by weshaggard — APPROVED** [review]
Looks good for java.

**Pull request merged**